Python/Sandhya/task2/store_in_sqlite.py

```diff
@@ -0,0 +1,20 @@
+import sqlite3
+import pandas
+from pandas import DataFrame
+
+con = sqlite3.connect('ScrapeData.db')
+# Saves database in same location as this file
+c = con.cursor()
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager